README.md

```diff
@@ -23,15 +25,37 @@ Terraform 0.12. Pin module version to ~> 2.0. Submit pull-requests to master bra
 
 Terraform 0.11. Pin module version to ~> 1.0. Submit pull-requests to terraform011 branch.
 
-## Usage
+### Usage for combined policy statements
+
+```hcl
+// TODO: make sure these references to module names are correct once pushed to the registry
+module "ou_scp" {
+  source = "trussworks/ou-scp"
+  target = aws_organizations_organizational_unit.root.id
```

Comment:
For your example, I'd use a more hypothetical OU like `aws_organizations_organizational_unit.my_ou.id` or something, just to keep people from doing something weird on accident.